src-web/containers/App.js

```diff
@@ -110,6 +111,10 @@ class App extends React.Component {
 
     const getSingleApplicationTabs = params => {
       const isArgoApp = _.get(location, 'search').indexOf('argoproj.io') !== -1
+      const isArgoAppset =
+        _.get(location, 'search').indexOf('&applicationset') !== -1
+      const isRemoteCluster =
+        _.get(location, 'search').indexOf('&cluster=') !== -1
```

Comment:
We should probably parse the query string here and then check for the keys/values we are interested in.

For example, searching for `&applicationset` means that applicationset cannot be the first query param, as in `?applicationset=foo`